vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -205,8 +205,10 @@ testKubeBinariesPresent() {
   1.19.13
   1.20.2-hotfix.20210310
   1.20.5-hotfix.20210322
+  1.20.7-hotfix.20210816
```

Comment:
so we weren't previously testing either of these versions in vhd content test? oops

Reply:
Yes, we were not and I was surprised by that too.